config/manager/kustomization.yaml

```diff
@@ -5,6 +5,12 @@ generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-- name: manager-config
-  envs:
+- envs:
   - controller_manager_config.properties
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
```

Comment:
Please use a `quay.io/project-flotta`-located image or remove `images`; maybe even revert the whole file.